test/SerialIOTest.cpp

```diff
@@ -3310,11 +3310,49 @@ iterate_nonstreaming_series( std::string const & file )
             auto iteration = iterations[ i ];
             auto E_x = iteration.meshes[ "E" ][ "x" ];
             E_x.resetDataset(
-                openPMD::Dataset( openPMD::Datatype::INT, { extent } ) );
+                openPMD::Dataset( openPMD::Datatype::INT, { 2, extent } ) );
             std::vector< int > data( extent, i );
-            E_x.storeChunk( data, { 0 }, { extent } );
-            // we encourage manually closing iterations, but it should not matter
-            // so let's do the switcharoo for this test
+            E_x.storeChunk( data, { 0, 0 }, { 1, extent } );
+            bool taskSupportedByBackend = true;
+            Span< int > span = E_x.storeChunk< int >(
+                { 1, 0 },
+                { 1, extent },
+                /*
+                 * Hijack the functor that is called for buffer creation if the
+                 * backend doesn't support the task to see whether the backend
+                 * did support it or not.
```

Comment:
Can you please split this into two sentences? It's a bit hard to parse for me :)

Reply:
Should we make this lambda a helper function? Looks like a generally useful implementation that people would use?

Reply:
> Can you please split this into two sentences? It's a bit hard to parse for me :)

Will do

> Should we make this lambda a helper function? Looks like a generally useful implementation that people would use?

I'm not sure about it.. This sidechannels the return value (the boolean) as a value caught-by-reference. Can we write this in a way that would actually be a good API?

Reply:
> Will do

Done